tests/secrets/test_secrets_backends.py

```diff
@@ -70,20 +69,6 @@ def test_connection_env_secrets_backend(self):
         # we could make this more precise by defining __eq__ method for Connection
         self.assertEqual(sample_conn_1.host.lower(), conn.host)
 
-    def test_connection_metastore_secrets_backend(self):
-        sample_conn_2a = SampleConn("sample_2", "A")
-        sample_conn_2b = SampleConn("sample_2", "B")
-        with create_session() as session:
-            session.add(sample_conn_2a.conn)
-            session.add(sample_conn_2b.conn)
-            session.commit()
-        metastore_backend = MetastoreBackend()
-        conn_list = metastore_backend.get_connections("sample_2")
-        host_list = {x.host for x in conn_list}
-        self.assertEqual(
-            {sample_conn_2a.host.lower(), sample_conn_2b.host.lower()}, set(host_list)
-        )
-
```

Comment:
I think we still need this test but maybe modify it to test one connection instead of two.